### PR TITLE
Update/raise-error-if-moids-not-found

### PR DESCRIPTION
**Pull request by jschoewe**
If no MOIDs were found for the given objects, then this action should fail. Otherwise this causes other actions in a workflow to fail because they are expecting an MOID when the this succeeds.

NOTE: If multiple objects are given and only one of them has an MOID then this action will still succeed. It only fails if no MOIDs are found

### CODE REVIEW

**Review by arm4b — COMMENTED** [review]
As usual, every change needs a corresponding CHANGELOG record and pack version bump

File: actions/get_moid.py [hunk 1]
```diff
@@ -43,4 +43,8 @@ def run(self, object_names, object_type, vsphere=None):
             except Exception as e:
                 self.logger.warning(str(e))
 
+        # Raise an error if no MOIDs were found for the VMs
```
Comment (nmaludy) [inline actions/get_moid.py:47]: instead of raising an exception you could just return (False, [])

**Review by nmaludy — CHANGES_REQUESTED** [review]

**Review by nmaludy — APPROVED** [review]
LGTM

**Review by nmaludy — CHANGES_REQUESTED** [review]
Actually, probably want to add a note into the description of the get_moid action saying that the action will fail if no hosts are found.

**Review by nmaludy — APPROVED** [review]
LGTM :+1:

**Pull request merged**